apps/server/src/modules/tool/external-tool/uc/external-tool.uc.ts

```diff
@@ -125,6 +135,14 @@ export class ExternalToolUc {
 		// Use secrets from existing config
 		const updatedConfigProps: ExternalToolConfig = { ...currentExternalTool.config, ...externalToolUpdateProps.config };
 
+		if (
+			ExternalTool.isLti11Config(updatedConfigProps) &&
```

Comment:
This does not work in cases where the type of the config was changed from basic to lti11. There is a validation that an oauth2 tool cannot change the config type (since we are too lazy to delete the hydra entry when the config type changes), but there is none for basic or lti11 (for now).